test/parallel/test-runner-coverage-thresholds.js

```diff
@@ -88,6 +88,25 @@ for (const coverage of coverages) {
     assert(!findCoverageFileForPid(result.pid));
   });
 
+  test(`test failing ${coverage.flag} with red color`, () => {
```

Comment:
hey @hpatel292-seneca, it seems this test is failing in the CI.
Could you please take a look at it?

Reply:
Hey @pmarchini, I tried reading logs on failed tests and it seems like this regex 
![image](https://github.com/user-attachments/assets/69f44e1a-83ff-4b0f-aa32-4026e3fd1588)
is not matching with the output. But as we are not logging anything I am not sure what we are getting output here.

But this test is running as expected and passing locally and in GitHub Action CI. 
Github Action run: https://github.com/nodejs/node/actions/runs/12320594158/job/34394971556
Below is the SS of the locally run test
![image](https://github.com/user-attachments/assets/39e99d70-e411-4aa7-975a-5a42c5e97434)

Reply:
https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/62632/testReport/junit/(root)/parallel/test_runner_coverage_thresholds/ <-- here one CI failure

Reply:
Yes, I saw this CI failure but I am not sure what is causing the failure as this test is running as expected on my local machine. I tried reading logs on CI but I can't find anything. Is there any way I can debug those CI runs??